saleor/vendor/models.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.db import models
+
+# Create your models here.
+from ..warehouse.models import Allocation, Warehouse
+
+
+class Vendor(models.Model):
+    """Model definition for Vendor."""
+
+    # TODO: Define fields here
+    user = models.OneToOneField(
+        settings.AUTH_USER_MODEL,
+        blank=True,
+        null=True,
+        related_name="vendor",
```

Comment:
related name should not conflit with other names.
https://docs.djangoproject.com/en/3.1/topics/db/models/#be-careful-with-related-name-and-related-query-name

Reply:
The documentation you point has shown the erroneous use of related name in case of child classes but here, our Vendor model doesn't have a child class. I don't think the problem translates to our case.